jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/auth/AuthConfigTest.java

```diff
@@ -36,7 +36,7 @@ void simpleConstructor() {
     // Since Base64.decodeBase64 handles URL-safe encoding, must explicitly check
     // the correct characters are used
     assertThat(config.toHeaderValue(new KitLogger.SilentLogger()))
-      .isEqualTo("eyJlbWFpbCI6InJvbGFuZEBqb2xva2lhLm9yZyIsInBhc3N3b3JkIjoiIz5zZWNyZXRzPz8iLCJ1c2VybmFtZSI6InJvbGFuZCJ9");
```

Comment:
This was changed because of https://github.com/eclipse/jkube/pull/2702/files#r1495937819.
Previously it read `{"email":"roland@jolokia.org","password":"#>secrets??","username":"roland"}` and now it reads `{"username":"roland","password":"#>secrets??","email":"roland@jolokia.org"}`